comps/guardrails/langchain/guardrails_tgi_gaudi.py

```diff
@@ -66,6 +79,7 @@ def safety_guard(input: TextDoc) -> TextDoc:
 
 if __name__ == "__main__":
     safety_guard_endpoint = os.getenv("SAFETY_GUARD_ENDPOINT", "http://localhost:8080")
+    safety_guard_model = os.getenv("SAFETY_GUARD_MODEL_ID", get_tgi_service_model_id(safety_guard_endpoint))
```

Comment:
Should the instructions be updated adding this environment variable?
https://github.com/opea-project/GenAIComps/tree/main/comps/guardrails#21-setup-environment-variables

Reply:
Thanks!

I added the following instructions:

https://github.com/opea-project/GenAIComps/blob/9ccabd113eeb973e4b817d7c22b32baf3f79cd2a/comps/guardrails/README.md?plain=1#L55-L61